incubator/java-openliberty/image/project/pom.xml

```diff
@@ -55,6 +55,14 @@
             </plugins>
         </pluginManagement>
         <plugins>
+            <!-- Enable the liberty-maven plugin. Doing this here in the parent pom
```

Comment:
I'd just collapse the pluginManagement a few lines above into this.. I can't think of how it'd be helpful to keep them separate.

Reply:
Done